pkg/model/chi/ch_config_generator.go

```diff
@@ -31,8 +31,9 @@ const (
 	// Special auto-generated clusters. Each of these clusters lay over all replicas in CHI
 	// 1. Cluster with one shard and all replicas. Used to duplicate data over all replicas.
 	// 2. Cluster with all shards (1 replica). Used to gather/scatter data over all replicas.
-	OneShardAllReplicasClusterName = "all-replicated"
-	AllShardsOneReplicaClusterName = "all-sharded"
+	OneShardAllReplicasClusterName  = "all-replicated"
+	AllShardsOneReplicaClusterName  = "all-sharded"
+	AllShardsAllClustersClusterName = "chi-%s"
```

Comment:
After internal discussion let's make it as
```
AllClustersAllShardsAndReplicasClusterName = "all-clusters"
```

Reply:
Updated.